code/game/machinery/doors/brigdoors.dm

```diff
@@ -201,10 +197,9 @@
 	if(stat & (NOPOWER|BROKEN))
 		return 0
 
-	if(!printed)
```

Comment:
Could you explain to me the reasoning for removing this flag? It appears to control redundant printing

Reply:
Fixed.